strapi/strapi
# ML: Add create folder flow

### What does it do?

Adds a "Create folder" button and a corresponding dialog to the media library.

<img width="1512" alt="Screenshot 2022-05-06 at 13 29 37" src="https://user-images.githubusercontent.com/2244375/167123344-1d5838b3-a4b6-4fc7-8bc3-04179ab937aa.png">


### Why is it needed?

To be able to create folders.

### How to test it?

1. Go to the media library
2. Click "Create folder" on the top right, see the network request (which fails for now, because the backend is not ready yet)


- SelectTree: https://github.com/strapi/strapi/pull/13073
